tutorials/benchmark_example.py

```diff
@@ -0,0 +1,134 @@
+r"""
+Benchmark Utility in PyLops-MPI
+=========================
+This tutorial demonstrates how to use the bencmark utility of PyLops-MPI. It contains various
```

Comment:
This tutorial demonstrates how to use the :py:func:`pylops_mpi.utils.benchmark` and :py:func:`pylops_mpi.utils.mark` utility methods in PyLops-MPI.